software/base/src/main/java/org/apache/brooklyn/entity/software/base/SoftwareProcessImpl.java

```diff
@@ -599,15 +561,67 @@ public void waitForEntityStart() {
      */
     @Override
     public final void start(final Collection<? extends Location> locations) {
+
```

Comment:
This method fix the type location for this entity, for example ssh or PaaS. So, it is not possible to start (migrate #924) the entity to a new kind of location.